lib/accelerometer.js

```diff
@@ -242,16 +305,38 @@ function Accelerometer(opts) {
     this.toGravity = opts.toGravity || function(raw) { return raw; };
   }
 
+  if (!this.onEnable) {
+    this.onEnable = function(on) {};
```

Comment:
Also, you can fake "enable"/"disable" for those that don't have hardware support by just returning from the dataHandler if state.enabled === false